site/app/models/Config.php

```diff
@@ -258,6 +261,10 @@ class Config extends AbstractModel {
     public function __construct(Core $core) {
         parent::__construct($core);
         $this->timezone = new \DateTimeZone($this->default_timezone);
+
+        // For now this will be set to 'MDY', and configured as a property of the Config class
+        // Eventually this should be moved to the User class and configured on a per-user basis
+        $this->date_time_format = new DateTimeFormat($this->core, 'MDY');
```

Comment:
Is there buy-in from @bmcutler that this is something we're going to support in the proposed fashion at some point in the future?